doc/lightweight-styling.md

```diff
@@ -0,0 +1,63 @@
+# Lightweight Styling
```

Comment:
This document seems to be a duplicate of https://github.com/unoplatform/uno.toolkit.ui/pull/713. This PR is the proper repo to integrate it, and I don't think the toolkit.ui one is appropriate. @kazo0 @agneszitte

Reply:
@rajamatt 
Confirmed with @jeromelaban 

- Uno.Themes
  - lightweight-styling.md
  - resources specific to Uno.Themes
- Uno.Toolkit.UI
  - lightweight-styling.md (which links to Uno.Themes lightweight-styling.md + Uno.Toolkit specific info)
  - resources specific to Uno.Toolkit.UI